Dev Tools/Sql/Report_KidSpring_Tag_and_Breakout.sql

```diff
@@ -0,0 +1,46 @@
+WITH KidSpringAttendeeGroups AS (
+	SELECT
+		g.Id
+	FROM
+		[Group] g
+	WHERE
+		g.GroupTypeId in (756, 757, 758, 759) 
```

Comment:
@benwiley Can you add comments to this Metric Value script to clarify what it's doing?  There's a header block I've put with the others you can reuse.